src/screens/Home/index.tsx

```diff
@@ -100,18 +73,19 @@ export default function Home() {
       },
     ],
   };
-  const handleTabChange = (index: number) => {
-    setActiveTab(index);
-  };
   return (
     <View>
-      {/* {renderTabView()} */}
-      <CustomTab
-        tabName={['Newsfeed', 'Explore']}
-        onTabChange={handleTabChange}
-      />
-      {renderTabComponent()}
-
+      <CustomTab tabName={['Newsfeed', 'Explore']} onTabChange={setActiveTab} />
+      {activeTab === 1 ? (
```

Comment:
it would be better to change `tab` to `enum` instead of number

e.g.
```
enum TabName = {
  TAB_1 = 'TAB_1'
}
```

Reply:
Thanks. updated krub :)